tests/test_message_segmenting.py

```diff
@@ -26,10 +26,42 @@ def test_message_profiling_gsm7(self, short_gsm7_text, long_gsm7_text, byte_stri
     def test_message_profiling_ucs2(self, short_ucs2_text, long_ucs2_text):
         profiled_message = profiling.profile_message(short_ucs2_text)
         assert profiled_message["num_segments"] == 1
-        assert profiled_message["message_length"] == 15
+        assert profiled_message["message_length"] == 14
```

Comment:
I didn't change this, but for some reason the existing tests failed. It currently fails on master.

Reply:
We made a late night fix previously and probably didn't run the test suite 🤦